Formula/fff.rb

```diff
@@ -0,0 +1,18 @@
+class Fff < Formula
+  desc ":rocket: fucking fast file-manager"
```

Comment:
Let's avoid emojis and expletives

Reply:
This exists(?)

https://github.com/Homebrew/homebrew-core/blob/000e81242ea0b17e213ed69a0796f5c8b7f31a1e/Formula/thefuck.rb

Reply:
It's the software name, not in the description. `Fast file manager` seems like an appropriate description.

Reply:
That then changes the name to `ffm`. I don't see the difference in whether the expletive is in the name of the program or the description. If anything its worse in the name since the command to install it will include `fuck` as opposed to the description.

By all means, make the changes required. I just don't like the selective enforcement of whatever guidelines and rules are in place for homebrew packages.